builtin/settingtypes.txt

```diff
@@ -532,6 +532,9 @@ pause_fps_max (FPS in pause menu) int 20
 #    View distance in nodes.
 viewing_range (Viewing range) int 100 20 4000
 
+#	Near plane distance in nodes.
```

Comment:
maybe add a note here that you should usually not need to change this value + the example you mention (=0.25 on mobile devices)